From: =?utf-8?q?R=C3=A9mi_Vanicat?= <vanicat@debian.org>
Date: Thu, 7 Apr 2016 16:07:07 +0200
Subject: Remove magit-version.el from autogenerated files.

For Debian build, magit-version.el come from the .tar.gz, and is not a
autogenerated file. This causes problem with the clean target.
---
 default.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/default.mk b/default.mk
index e493959..a7a5d1c 100644
--- a/default.mk
+++ b/default.mk
@@ -97,7 +97,7 @@ ELS += magit-imenu.el
 ELS += magit-bookmark.el
 ELCS = $(ELS:.el=.elc)
 ELMS = magit.el $(filter-out $(addsuffix .el,$(PACKAGES)),$(ELS))
-ELGS = magit-autoloads.el magit-version.el
+ELGS = magit-autoloads.el
 
 ## Versions ##########################################################
 
